src/internal/operators/combineLatestWith.ts

```diff
@@ -80,13 +81,19 @@ export function combineLatest<T, TOther, R>(
 /**
  * @deprecated Deprecated, use {@link combineLatestWith} or static {@link combineLatest}
  */
-export function combineLatest<T, R>(...args: (ObservableInput<any> | ((...values: any[]) => R))[]): OperatorFunction<T, R> {
+export function combineLatest<T, R>(...args: (ObservableInput<any> | ((...values: any[]) => R))[]): OperatorFunction<T, unknown> {
```

Comment:
nitpick: `R` is used in this signature's parameter types, so it could remain in the return type, but if using `unknown` instead of `R` solves some problem, I guess that's fine. The general rule is that type parameters [have to be used twice](https://effectivetypescript.com/2020/08/12/generics-golden-rule/) - to avoid their being pointless.

Reply:
It's `unknown` because it all depends on whether or not the user passed the deprecated resultSelector or not. And I'm too lazy to dance around the types and cast etc for this internal bit. I didn't think it added anything to do so.